src/many-ledger/src/main.rs

```diff
@@ -1,7 +1,8 @@
 use clap::Parser;
 use many_identity::verifiers::AnonymousVerifier;
 #[cfg(feature = "balance_testing")]
-use many_identity::{Address, Identity};
+use many_identity::Address;
+use many_identity::Identity;
```

Comment:
```suggestion
use many_identity::{Address, Identity};
```

Reply:
So I merged it based on the suggestion but I remember why I did this; there's a feature guard one line above.

`Address` is only used with `balance_testing` while `Identity` is required all the time.